arviz/data/io_cmdstanpy.py

```diff
@@ -393,6 +438,105 @@ def to_inference_data(self):
             },
         )
 
+    @requires("posterior")
+    def posterior_to_xarray_v68(self):
```

Comment:
Could we change the logic so this is "default" and the other function is "old"?

Reply:
yes!